octobot_trading/exchanges/data/exchange_config_data.py

```diff
@@ -107,7 +107,7 @@ def _set_config_time_frame(self):
                 self.traded_time_frames.append(time_frame)
         # add shortest timeframe for realtime evaluators
         client_shortest_time_frame = find_min_time_frame(
-            self.exchange_manager.client_time_frames[CONFIG_WILDCARD], MIN_EVAL_TIME_FRAME)
+            next(iter(self.exchange_manager.client_time_frames.values())), MIN_EVAL_TIME_FRAME)
```

Comment:
Why the first `symbol` or `WILDCARD` key should be the shortest time frame ?

Reply:
Since we are in an exchange specific situation, any symbol should have the same available time frames. Exceptions being for backtesting with different importers on the same exchange and for some reason different time frames: in this case we don't need to add the shortest timeframe for visualisation because backtesting is fast.

Reply:
So if you think there is no need of of dictionnary for `client_time_frames` but a list with the `time_frame` is enough we should go for it.